packaging/leapp-repository.spec

```
@@ -22,25 +36,35 @@ URL:            https://oamg.github.io/leapp/
 Source0:        https://github.com/oamg/%{name}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 Source1:        deps-pkgs.tar.gz
 BuildArch:      noarch
+
+%if 0%{?rhel} && 0%{?rhel} == 7
```

```suggestion
%if 0%{?rhel} == 7
```
is enough, because 0 == 7 is false anyway but that's not needed to be fixed now.

---

I will fix it probably in upcoming PR where I will finalize the packaging completely.